Create LOCATIONS.TXT

vpnarea opened the pull request:
Locations of VPNArea.com servers

Zomboided commented:
TEMPLATE.txt is fine now, but this should be called LOCATIONS.txt
Also, can you make the locations more readable/consistent/easier to read - It doesn't matter, but I spend a bunch of time doing this when adding providers to avoid random looking strings - so for example "San Francisco" rather than the selection you have, "Adelaide 2", and loose stuff like 'NEW' and the names of the IP services (because who cares).  Users can switch to see what the IP is they're actually using anyway.

You've also missed a few other updates that will actually make this work but I'm changing those files at the moment so if this is ready when I do the update I'll add it all in and release it.

vpnarea commented:
Just updated the locations and tidied up their names. I can't remove the numbers from all of them because we have multiple servers in a single location, such as Adelaide, but I have removed "NEW" from all of the servers. I can't also replace the IPs with hostnames because we don't use hostnames for the "remote" directive in configs. We avoid this to prevent government firewalls from easily targeting our servers. 

Please let me know if anything else has to be changed.
